elastic/kibana
# [Ingest Pipelines] UI improvements to list view

Closes https://github.com/elastic/kibana/issues/179508

## Summary

This PR slightly updates the UI for the list view of ingest pipelines.

##### How to test

* Start up kibana and es
* Navigate to Stack management -> Ingest pipelines
* Verify that:
  * Visuals are updated
  * Sorting works
  * Filtering and reload works 


<details>
<summary>Screenshots</summary>

// Empty state
![Screenshot 2024-03-28 at 09 39 21](https://github.com/elastic/kibana/assets/1191206/c7b21342-123d-4bb6-bce1-c3d315465a70)

// New list view
![Screenshot 2024-03-28 at 09 40 37](https://github.com/elastic/kibana/assets/1191206/e62f3ad0-ae33-4fe8-a7e4-97da7a20ea89)

</details>
